tests/lax_scipy_test.py

```diff
@@ -405,6 +454,137 @@ def testSphHarmCornerCaseWithWrongNmax(self):
 
     self.assertAllClose(actual, expected, rtol=1e-8, atol=9e-5)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {'testcase_name':
+        '_n_zero_sv={}_degeneracy={}_geometric_spectrum={}'
+        '_max_sv={}_shape={}_method={}_side={}'
+        '_nonzero_condition_number={}_dtype={}_seed={}'.format(
+          n_zero_sv, degeneracy, geometric_spectrum, max_sv,
+          shape, method, side, nonzero_condition_number, dtype,
+          seed
+        ),
+        'n_zero_sv': n_zero_sv, 'degeneracy': degeneracy,
+        'geometric_spectrum': geometric_spectrum,
+        'max_sv': max_sv, 'shape': shape, 'method': method,
+        'side': side, 'nonzero_condition_number': nonzero_condition_number,
+        'dtype': dtype, 'seed': seed}
+      for n_zero_sv in n_zero_svs
+      for degeneracy in degeneracies
+      for geometric_spectrum in geometric_spectra
+      for max_sv in max_svs
+      for shape in polar_shapes
+      for method in methods
+      for side in sides
+      for nonzero_condition_number in nonzero_condition_numbers
+      for dtype in jtu.dtypes.all_floating
+      for seed in seeds))
+  def testPolar(
+    self, n_zero_sv, degeneracy, geometric_spectrum, max_sv, shape, method,
+      side, nonzero_condition_number, dtype, seed):
+    """ Tests jax.scipy.linalg.polar."""
+    np.random.seed(seed)
+    matrix, _ = _initialize_polar_test(
+      shape, n_zero_sv, degeneracy, geometric_spectrum, max_sv,
+      nonzero_condition_number, dtype)
+    if dtype in (jnp.bfloat16, np.float16):
+      self.assertRaises(
+        NotImplementedError, jsp.linalg.polar, matrix, method=method,
+        side=side)
+      return
+
+    unitary, posdef, info = jsp.linalg.polar(matrix, method=method, side=side)
```

Comment:
I'm actually missing where we export `polar` in the `jax.scipy.linalg` namespace. Don't we need to add it to the list in `jax/scipy/linalg.py`

Reply:
isn't it in there?